app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -17,7 +17,7 @@
 import java.io.IOException;
 
 /**
- * Simple login activity class.
+ * Draws the screen is used to login by entering a password
```

Comment:
Class JavaDoc should make sense without the class itself, so could you start with "{@link Activity} that..."

Also the text isn't correct English